apps/web/src/pages/integrations/components/UpdateIntegrationSidebarHeader.tsx

```diff
@@ -36,7 +36,7 @@ export const UpdateIntegrationSidebarHeader = ({
 
     const { channel: selectedChannel, environmentId, integrationId, primary } = provider;
     const hasSameChannelActiveIntegration = !!providers
-      .filter((el) => !NOVU_PROVIDERS.includes(el.providerId) && el.integrationId !== integrationId)
```

Comment:
This was missed when we moved the Novu integrations to DB. We do allow selecting Novu integrations as a primary integration.